bundles/org.dataflowanalysis.converter/META-INF/MANIFEST.MF

```diff
@@ -5,8 +5,14 @@ Bundle-SymbolicName: org.dataflowanalysis.converter
 Bundle-Version: 4.0.0.qualifier
 Export-Package: 
  org.dataflowanalysis.converter,
- org.dataflowanalysis.converter.microsecend,
- org.dataflowanalysis.converter.webdfd
+ org.dataflowanalysis.converter.chain,
```

Comment:
`converter.chain` yields an error for not existing

Reply:
Fixed in 2b4b018f877f